doc/examples/jest+react/link.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import axe from 'axe-core';
+import { mountToDoc } from './test-helpers';
+
+import Link from './Link';
+
+test('Link has no aXe violations', (done) => {
+  const linkComponent = mountToDoc(
+    <Link page="http://www.axe-core.org">aXe website</Link>
+  );
+  const linkNode = linkComponent.getDOMNode();
+
+  axe.run(linkNode, (err, { violations }) => {
```

Comment:
axe drops those into incomplete I believe. But we can make it explicit if you think that's better?